modules/ztp-configuring-disk-partitioning.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-configuring-disk-partitioning_{context}"]
+= Configuring disk partitioning with SiteConfig
+
+You can use the `SiteConfig` YAML file to generate the `MachineConfig` CR used for disk partitioning. You need to to modify values in the `MachineConfig` CR to reflect dependencies on the underlying disk.
+
+[NOTE]
+====
+You must use persistent naming for devices to avoid device names such as `/dev/sda` and `/dev/sdb` being switched at every reboot. You can use `rootDeviceHints` to choose the bootable device and then use same device for further partitioning: in this case, for Image registry.
+====
+
+.Prerequisites
+
+* You configured the hub cluster for generating the required installation and policy CRs.
+
+* You created a Git repository where you manage your custom site configuration data. The repository must be accessible from the hub cluster and be defined as a source repository for the Argo CD application.
+
+.Procedure
+
+. Add the following YAML to the `SiteConfig.yaml` file to generate `MachineConfig` for disk partitioning. :
++
+[source,yaml]
+----
+#Comment ....
+nodes:
+  - rootDeviceHints:
+      wwn: "0x62cea7f05c98c2002708a0a22ff480ea"
+    diskPartition:
+    - device: /dev/disk/by-id/wwn-0x62cea7f05c98c2002708a0a22ff480ea <1>
+      partitions:
+       - mount_point: /var/imageregistry
+         size: 102500 <2>
+         start: 344844 <3>
+----
+<1> Depends on the hardware. The setting can also be a serial number or device name, for example `wwn`. The value must match with the `win` entry for `rootDeviceHint`.
```

Comment:
1 - the example is already using wwn (remove "for example `wwn`")
2- "The value must match with the `win` entry for `rootDeviceHint`" change to "The value must match with the entry for `rootDeviceHint`" (drop 'win')

Reply:
Done.